### ISSUE TITLE
Metrics: Make PROMETHEUS_MONITORING_GROUP true by default in Weaviate

### ISSUE THREAD

**Issue by StefanBogdan** [weaviate/weaviate#3458]
Would be great for it to be the default one so if anyone forgets to enable it, they would not have to search the docs for ENV Vars to understand how to enable it.

Also it is not yet listed in the Helm chart as well so the more reasons to have it `true` by defaul.

**Comment by etiennedi** [comment]
Agreed, this has quickly become the preferred setup since it was introduced as an opt-in feature. Now might be the right time to make it opt-out. Unless anyone has any objections let's make the switch in `1.22`.
